benchmark/version.txt

```diff
@@ -1 +1 @@
-0.13.1-SNAPSHOT
```

Comment:
Is this supposed to go up to 0.22? Or is it versioned independently and should go up to 0.14?